includes/class-salesforce.php

```diff
@@ -41,13 +41,41 @@ class Salesforce {
 	 * Add hooks.
 	 */
 	public static function init() {
+		add_action( 'init', [ __CLASS__, 'platform_check' ] );
 		add_action( 'rest_api_init', [ __CLASS__, 'register_api_endpoints' ] );
 		add_action( 'admin_enqueue_scripts', [ __CLASS__, 'register_admin_scripts' ] );
 		add_action( 'woocommerce_order_actions_start', [ __CLASS__, 'wc_order_show_sync_status' ] );
 		add_filter( 'woocommerce_order_actions', [ __CLASS__, 'wc_order_manual_sync' ] );
 		add_action( 'woocommerce_order_action_newspack_sync_order_to_salesforce', [ __CLASS__, 'wc_order_manual_sync_action' ] );
 	}
 
+	/**
+	 * Check if Newspack is the current Reader Revenue platform.
+	 * If not, disable syncing. If so and the required webhook
+	 * doesn't exist, create it.
+	 */
+	public static function platform_check() {
+		$is_newspack = self::platform_is_newspack();
```

Comment:
The method is already implemented in `Donations` class.

```suggestion
		$is_newspack = Donations::is_platform_wc();
```

(this will have to change in a few more places here)

Reply:
Nice, I missed that. Updated in b591fde